### PR TITLE
Fp ts 2

### PR DESCRIPTION
**Pull request by raveclassic**

### CODE REVIEW

File: src/swagger.ts [hunk 1]
```diff
@@ -19,20 +17,26 @@ export type TContactObject = {
...
-	url: stringOption,
-	email: stringOption,
-});
+export const ContactObject = t.type(
+	{
+		name: stringOption,
+		url: stringOption,
+		email: stringOption,
+	},
+	'TContactObject',
```
Comment (scink) [inline src/swagger.ts]: do we need `T` prefix?

Comment (raveclassic) [reply src/swagger.ts]: We can change it in a separate `refactoring` PR.

Comment (raveclassic) [reply src/swagger.ts]: @scink https://github.com/devexperts/swagger-codegen-ts/issues/43

**Comment by scink** [comment]
looks good for me, thank!

**Pull request merged**